homeassistant/components/nest/local_auth.py

```diff
@@ -11,7 +11,8 @@
 def initialize(hass, client_id, client_secret):
     """Initialize a local auth provider."""
     config_flow.register_flow_implementation(
-        hass, DOMAIN, 'local', partial(generate_auth_url, client_id),
+        hass, DOMAIN, 'configuration.yaml',
```

Comment:
Sneaking in a small change to how the entry for nest is called, it was confusing.